.github/workflows/wheels.yml

```diff
@@ -52,185 +61,160 @@ jobs:
-          CIBW_BEFORE_ALL_MACOS: cp -r `pwd`/hdf5_build /tmp/hdf5
-          # Keep in sync with "Prerequisites" in User's Guide.
-          CIBW_BEFORE_ALL_LINUX: >
-            cp -r `pwd`/hdf5_build /tmp/hdf5 &&
-            yum -y update &&
-            yum install -y zlib-devel bzip2-devel lzo-devel
-          CIBW_REPAIR_WHEEL_COMMAND_MACOS: >
-            DYLD_FALLBACK_LIBRARY_PATH=/tmp/hdf5/lib delocate-listdeps {wheel} &&
-            DYLD_FALLBACK_LIBRARY_PATH=/tmp/hdf5/lib delocate-wheel --require-archs {delocate_archs} -w {dest_dir} {wheel}
-          CIBW_TEST_COMMAND: "python -m tables.tests.test_all"
-          CIBW_TEST_SKIP: "*_aarch64 *_arm64"
+          CIBW_BUILD: '${{ matrix.build }}'
+          CIBW_ARCHS: '${{ matrix.arch }}'
 
       - name: Copy requirements.txt
         run: |
           cp requirements.txt ./wheelhouse/
 
-      - uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+      - uses: actions/upload-artifact@v4
         with:
           path: |
             ./wheelhouse/*.whl
             ./wheelhouse/*.txt
-
-  build_wheels_windows:
-    name: Build wheels for ${{matrix.arch}} on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [windows-latest]
-        arch: [win_amd64]
-
-    steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-        with:
-          fetch-depth: 0
-          submodules: true
-
-      - uses: actions/setup-python@65d7f2d534ac1bc67fcd62888c5f4f3d2cb2b236 # v4.7.1
-        name: Install Python
-        with:
-          python-version: 3.x
-
-      - name: Install Miniconda
-        uses: conda-incubator/setup-miniconda@3b0f2504dd76ef23b6d31f291f4913fb60ab5ff3 # v2.2.0
-        with:
-          channels: defaults,conda-forge
-          use-only-tar-bz2: true
-
-      - name: Install cibuildwheel
-        run: |
-          python -m pip install --require-hashes -r ./.github/workflows/requirements/wheel-requirements.txt
-
-      - name: Build wheels for Windows (${{ matrix.arch }})
-        run: cibuildwheel --output-dir wheelhouse
-        env:
-          # Keep Python in sync with ``project.classifiers`` in ``pyproject.toml``.
-          CIBW_BUILD: "cp39-${{ matrix.arch }} cp310-${{ matrix.arch }} cp311-${{ matrix.arch }} cp312-${{ matrix.arch }}"
-          # Keep in sync with "Prerequisites" in User's Guide.
-          CIBW_BEFORE_ALL_WINDOWS: >
-            conda create --yes --name=build &&
-            conda activate build &&
-            conda config --env --set subdir ${{ matrix.arch == 'win32' && 'win-32' || 'win-64' }} &&
-            conda install --yes blosc bzip2 cython hdf5 lz4 lzo snappy zstd zlib
-          CIBW_ENVIRONMENT_WINDOWS: 'CONDA_PREFIX="C:\\Miniconda\\envs\\build" PATH="$PATH;C:\\Miniconda\\envs\\build\\Library\\bin"'
-          CIBW_ENVIRONMENT: "PYTABLES_NO_EMBEDDED_LIBS=true DISABLE_AVX2=true"
```

Comment:
Pretty sure the `CIBW_ENVIRONMENT` vars here weren't being used because the `CIBW_ENVIRONMENT_WINDOWS` was also defined.

If I set `PYTABLES_NO_EMBEDDED_LIBS=true` it fails, presumably because `blosc2` isn't in the build deps? In any case, now `PYTABLES_NO_EMBEDDED_LIBS` isn't used anywhere, and locally `cibuildwheel` on Windows works so hopefully it's okay.